### PR TITLE
Dom0 kernel installer transformer changes as per new Dom0 VHD

### PR DESCRIPTION
**Pull request by smit-gardhariya**
This PR will make changes in dom0 kernel installer transformers (dom0 and dom0_binaries) as per new VHD changes.

### CODE REVIEW

File: lisa/transformers/dom0_kernel_installer.py [hunk 1]
```diff
@@ -187,28 +172,28 @@ def _copy_kernel_binary(
...
     node: Node,
     is_initrd: bool,
     current_kernel: str,
     new_kernel: str,
 ) -> None:
-    ll_conf_file: str = "/boot/efi/linuxloader.conf"
+    ll_conf_file: str = "/boot/mariner.cfg"
     sed = node.tools[Sed]
 
     # Modify the linuxloader.conf to point new kernel binary
```
Comment (pupacha) [inline lisa/transformers/dom0_kernel_installer.py]: Please update this comment accordingly.

File: lisa/transformers/dom0_kernel_installer.py [hunk 2]
```diff
@@ -187,28 +172,28 @@ def _copy_kernel_binary(
...
-        regexp=f"KERNEL_PATH=vmlinuz-{current_kernel}",
-        replacement=f"KERNEL_PATH=vmlinuz-{new_kernel}",
+        regexp=f"mariner_linux=vmlinuz-{current_kernel}",
+        replacement=f"mariner_linux=vmlinuz-{new_kernel}",
         file=ll_conf_file,
         sudo=True,
     )
 
     if is_initrd:
         # Modify the linuxloader.conf to point new initrd binary
```
Comment (pupacha) [inline lisa/transformers/dom0_kernel_installer.py]: Same here.

File: lisa/transformers/dom0_kernel_installer.py (hunk 1 above)
Comment (smit-gardhariya) [reply lisa/transformers/dom0_kernel_installer.py]: @pupacha Thnx for pointing this. It is done. Please check now

File: lisa/transformers/dom0_kernel_installer.py (hunk 2 above)
Comment (smit-gardhariya) [reply lisa/transformers/dom0_kernel_installer.py]: @pupacha Thnx for pointing this. It is done. Please check now

**Review by pupacha — APPROVED** [review]

**Review by LiliDeng — APPROVED** [review]

**Pull request merged**